GeofenceDisplay/src/geofence-display.ts

```diff
@@ -1,155 +1,111 @@
-export {}
-
 import L from 'leaflet'
+// import Gubu from 'gubu'
 import './geofence-display.css'
+// import { RasterCoords } from './rastercoords'
+
+// TODO: implement Gubu with interfaces
+interface Geofence {
```

Comment:
since we have class called Geofence, have a separate type will cause confusion
maybe rename to `GeofenceDef` as it a definition

Reply:
Changed to `interface GeofenceDef`